### DISCUSSION TITLE
How to create custom travel routes?

### DISCUSSION THREAD

**Discussion by mcbaer** [Courseplay/Courseplay_FS22#570] [Q&A] [answered]
I just bought FS22, after seeing that courseplay is available (in beta) for the game. After adding CP to the game, I'm very lose at this point.  None of the key combination I remember from FS19 are working. I can't seem to find where I can create my own routes for traveling from one location to another. 

I do see some options in the ESC Menu, however still no option to create custom travel routes. Just options to set a destination, and the vehicle goes there. While this is all good, as it saves the player time from having to set up routes, the issue is the routes the tractor takes are broken. 

With the "pre-configured" (sorry don't know what to call them) the tractor zig zags around, stops often like its thinking what it needs to do next, when going from a 2 lane road to a 1 lane for whatever reason it will move from the far left lane, to the right lane to then just stop for a moment then shift back to the left to continue onto the 1 lane road. Also on Elm Creek there is a very very very badly designed bridge by GIANTs map designer, that when you travel over the incline it often sends the vehicle into the air, several times my tractor has tipped over, so I like to avoid this bridge even if it means driving a bit further, however with CP I can't do this as the pre-configured routes won't avoid it. 

Is there no plan to allow for custom travel routes? with CP for FS22 will we be forced/required to use only the pre-configured route? I'd like to know the expected roadmap for the customer travel routes, as I only have a shot time before I can request a refund for FS22 on Steam. Why a refund? Well because I don't like playing FS without courseplay. I often have the game open while I'm working (on a different computer) so CP comes in handy to let me focus for a min on work while not losing any effort in the game. I pretty much only ever used CP for recording travel/delivery courses, so I could set equipment on their way to a field while I needed to focus on work for a moment and I like to do the actual fieldwork myself w/o the help of CP, its just all the driving around the map that bores me as everything is so spread out.. Anyways, without being able to make custom travel routes, I find FS22 would not be as enjoyable and don't want to wast the $90 spent if I can't enjoy it. 

If the option to create custom travel routes is in the beta, and I've some how missed it, I apologize for the post. I clicked around for some time trying to figure out, but this new way of using CP is very confusing. Much preferred the click to start recording route / click to stop recording method of things. And not having to open up the ESC menu to do something, as it often has to pause for a moment to save the game.

**Comment by Redster999** [comment, marked as answer]
You should really read up on the forum but I will try to sum it up for you. CP for fs22 is very different. There are things yet to come to CP but route creation is not one of them. The reason why is that the CP team wants to focus on fieldwork and combining their mod together with Autodrive (AD) Which is another mod that does a very good job of these routes your are looking for. CP and AD team are now working together. If you have tried this combi before it was very good in FS19. Now all we have to do is wait for the merging. Meanwhile you should try out Autodrive. Once you get the concept, you will understand why they are doing this. And why I think it will be better than ever.

And yes, the original helpers should quit their job

**Reply by mcbaer** [reply]
Thanks. The search for CP FS22 leads here, so this is what I used vs a forum elsewhere. Plus, I wouldn't spend time signing up for a forum, if there wasn't the discussions tab in GitHub I would have just left it at that and uninstalled/refunded the game then.

Anywho, thanks for the insight into my concern. I had heard of the AutoDrive before, for FS19, but gave no thought to trying it out as I was already using CoursePlay which already did what I needed. I saw no purpose in installing/running another mod to do basically the same thing. I'll give it a go however, seeing that its currently the only options for the function/feature I'm looking for, and just remove CP as I this point I don't have a need for it. 

Again thanks for your response

**Reply by Redster999** [reply]
Glad to help. Sorry when I said forum I rally meant the main page and the issues. There is little to no info on CP besides this page. There is bogus pages but here is where new gets out first.

I thought the same you did. Why AD when i can create routes with cp. Well the difference is huge though. Using AD you never have to create the same route twice. And in fs 19 the tractor and seeder will automatic get more seeds when empty if you set up the route. Ad wil also go get gas or bring by vehigle to the repair shop when needed. As long as its on the route you created. 

Think of the routes as train tracks. You can combine the whole map and just choose your destination to fill and/or load. 

The current version of cp really handles the fields that are not square. Helpers sometimes stop on these fields. Or it also lets you create bigger fields from scrap, like in the map No mans land. 

Try it and learn it. It wil only get better. There has been 19 releases since christmas eve. They are working hard on this
